vendor/engines/split_accounts/app/services/split_accounts/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,52 @@
+module SplitAccounts
+  module Converters
+    class OrderDetailToJournalRows
+
+      SplitAmount = Struct.new(:split, :amount)
+
+      attr_accessor :journal, :order_detail, :total, :splits
+
+      def initialize(journal, order_detail, options = {})
+        @journal = journal
+        @order_detail = order_detail
+        @total = options[:total] || order_detail.total
```

Comment:
Are these options primarily for testing? They feel a little weird to make them overridable in actual work.